NBXplorer.Client/ExplorerClient.cs

```diff
@@ -322,6 +322,16 @@ public Task<GetBalanceResponse> GetBalanceAsync(DerivationStrategyBase userDeriv
 			return SendAsync<GetBalanceResponse>(HttpMethod.Get, null, "v1/cryptos/{0}/derivations/{1}/balance", new[] { CryptoCode, userDerivationScheme.ToString() }, cancellation);
 		}
 
+
+		public GetBalanceResponse GetBalance(string address)
+		{
+			return GetBalanceAsync(address).GetAwaiter().GetResult();
+		}
+		public Task<GetBalanceResponse> GetBalanceAsync(string address, CancellationToken cancellation = default)
+		{
+			return SendAsync<GetBalanceResponse>(HttpMethod.Get, null, "v1/cryptos/{0}/addresses/{1}/balance", new[] { CryptoCode, address }, cancellation);
```

Comment:
Reuse `GetBalanceAsync(DerivationStrategyBase userDerivationScheme)` instead, there is a subclass to pass address only.

Reply:
I've only seen logic with TrackedSource when address was used. Still don't understand how to create DerivationStrategy from factory for btc address

Reply:
Ah you are right this is a tracked source, not a derivation strategy.